File: Modulos/Configuracion/Configuracion.Dominio/Modelo/ConsultaParametrosTablasDefinidas.cs
using Infraestructura.Core.Comun.Presentacion;

namespace Configuracion.Dominio.Modelo
{
    public class ConsultaParametrosTablasDefinidas: Consulta
    {
        public int IdTabla { get; set; }
        public string Nombre { get; set; }
        public decimal? IdParametro { get; set; }
        public bool IncluirDadosDeBaja { get; set; }
    }
}

File: Web/Source/app/soporte/modelo/consulta-parametros-tablas-definidas.ts
export class ConsultaParametrosTablasDefinidas {
  public idTabla: number;
  public nombre: string;
  public idParametro: number;
  public incluirDadosDeBaja: boolean = false;
  public numeroPagina: number;
  public tamañoPagina: number;

  constructor(idTabla?: number,
              filtroNombre?: string,
              idParametro?: number,
              incluirDadosDeBaja?: boolean,
              numeroPagina?: number,
              tamañoPagina?: number) {
    this.idTabla = idTabla;
    this.nombre = filtroNombre;
    this.idParametro = this.idParametro;
    this.incluirDadosDeBaja = incluirDadosDeBaja;
    this.numeroPagina = numeroPagina;
    this.tamañoPagina = tamañoPagina;
  }
}
